frontend/src/metabase/static-viz/components/XYChart/types.ts

```diff
@@ -111,6 +115,6 @@ export interface XScale<T = any> {
     ? ScaleTime<number, number, never>
     : ScaleLinear<number, number, never>;
   bandwidth?: number;
-  lineAccessor: XYAccessor<SeriesDatum>;
-  barAccessor?: XYAccessor<SeriesDatum>;
+  lineAccessor: DatumAccessor;
```

Comment:
Fix the type since accessors created in `scale.ts` e.g.
https://github.com/metabase/metabase/blob/0a0849b2a50b8a490299a1c774aeb8635fb021dd/frontend/src/metabase/static-viz/components/XYChart/utils/scales.ts#L48-L55

won't accept the second argument `flipped`, that's only used in `<Values />` component, and the type definition is now moved there.